cluster-provision/okd/hacks/release-4.4

```diff
@@ -1,34 +1,34 @@
 diff --git a/cmd/openshift-install/create.go b/cmd/openshift-install/create.go
-index f9ae4c6bb..dea45f0d7 100644
+index 70a8201a8..e44a4d03e 100644
 --- a/cmd/openshift-install/create.go
 +++ b/cmd/openshift-install/create.go
-@@ -244,7 +244,7 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config, director
+@@ -252,7 +252,7 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config, director
  
  	discovery := client.Discovery()
  
--	apiTimeout := 30 * time.Minute
+-	apiTimeout := 20 * time.Minute
```

Comment:
we clone the openshift installer (create go comes from there), and patch the changes,
one of the issues that provision didnt work is that the code is changed and it couldnt be applied (and then exit with an error directly, in the very beginning)

it doesnt change a lot, this was what we are doing always, in order to extend timeout and cert expiration


all i did is what i wrote in the previous comment
"applied all changes and fixed all conflicts and then recreated a diff file which is this one
so its there already in the code, else it would reject it"

the actual time after the applying is 120 minutes